frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.html

```diff
@@ -28,26 +28,34 @@ <h5 class="fw-light">Leaderboard</h5>
           >
           </app-selectphase>
         </div>
-        <div class="leaderboard-precision" *ngIf="leaderboard.length > 0 && selectedPhaseSplit != null">
-          <span class="spacer"></span>
-          <span class="fw-regular">Leaderboard precision</span>
-          <div class="rangeslider fa-stack fa-2x">
-            <mat-slider
-              thumbLabel
-              min="0"
-              max="5"
-              step="1"
-              thumbLabel
-              placeholder="leaderboardPrecisionValue"
-              [(ngModel)]="leaderboardPrecisionValue"
-              (change)="updateLeaderboardDecimalPrecision($event)"
-            >
-            </mat-slider>
+      </div>
+      <br />
+      <div *ngIf="leaderboard.length > 0 && selectedPhaseSplit != null">
+        <div class="row row-lr-margin">
+          <div class="col-sm-12 col-xs-12 col-lr-pad" class="leaderboard-precison">
+            <span class="fw-semibold">
+              Leaderboard precision value: &nbsp; <mat-chip>{{ leaderboardPrecisionValue }}</mat-chip></span
+            ><br />
+            <span class="fw-light"
+              >Changing the precision value will show the results with that precision value to the participants.
```

Comment:
Minor change: @Kajol-Kumari Can you please format the code of this file changes like this one here ^